installinator/src/dispatch.rs

```diff
@@ -103,47 +111,97 @@ struct InstallOpts {
 
 impl InstallOpts {
     async fn exec(self, log: slog::Logger) -> Result<()> {
-        // Assume that buffering up the entire artifact in memory is OK, because we expect to have
-        // much more RAM available than the size of the host image. (If this changes in the future,
-        // we'll need to have some sort of streaming here.)
-        let artifact = FetchedArtifact::loop_fetch_from_peers(
+        let artifact_hashes = self.artifact_ids.resolve()?;
+
+        let host_phase_2_id = ArtifactHashId {
+            // TODO: currently we're assuming that wicket will unpack the host
+            // phase 2 image. We may instead have the installinator do it.
+            // kind: KnownArtifactKind::Host.into(),
+            kind: ArtifactKind::HOST_PHASE_2,
```

Comment:
Where is this defined?

Reply:
This is defined in https://github.com/oxidecomputer/omicron/pull/2387.

Reply:
Thanks!